src/components/file-system-explorer/index.tsx

```diff
@@ -70,13 +71,22 @@ const FileSystemExplorer: FC<FileSystemExplorerProps> = ({
   const contentRef = useRef<HTMLDivElement>();
 
   const [columns, setColumns] = useState(data);
+  const [isEmptyFolder, setIsEmptyFolder] = useState<boolean>(false);
 
   useEffect(() => {
-    setColumns(data);
+    if (Array.isArray(data)) {
+      if (Array.isArray(data[data.length - 1]) && data[data.length - 1].length === 0) {
+        setIsEmptyFolder(true);
+        setColumns(data.slice(0, -1));
+      } else {
+        setIsEmptyFolder(false);
+        setColumns(data);
+      }
+    }
   }, [data]);
 
   const [activeFile, setActiveFile] = useState<ActiveFile>();
-  //@ts-ignore
+  // @ts-ignore
```

Comment:
what do we need this `@ts-ignore` here for?

Reply:
Not sure, it was there before, here just added a space to escape the eslint error msg. 
Without it there seems no differences, should I remove it?